packages/docs/content/foundation/theme-colors.mdx

```diff
@@ -101,12 +112,7 @@ These colors are used largely for background blocks and large content areas. Whe
 <ColorSwatchList
   colorNames={[
     'background',
-    'background-inverse',
```

Comment:
Removed gray colors from this to make space for more intentional color naming in the future